pandas/tests/plotting/test_misc.py

```diff
@@ -308,3 +308,26 @@ def test_get_standard_colors_random_seed(self):
         color1 = _get_standard_colors(1, color_type='random')
         color2 = _get_standard_colors(1, color_type='random')
         assert color1 == color2
+
+    def test_get_standard_colors_no_appending(self):
+        # GH20726
+
+        # Make sure not to add more colors so that matplotlib can cycle
+        # correctly.
+        from pandas.plotting._style import _get_standard_colors
```

Comment:
Make these module level imports

Reply:
Done. However there are a lot of other local import in this file and many other tests, maybe it's time for some clean up?